tools/distpartitioning/utils.py

```diff
@@ -63,6 +63,33 @@ def read_json(json_file):
 
     return val
 
+def get_etype_featnames(etype_name, schema_map):
+    """
+    Retrieves edge feature names for a given edge_type
+
+    Parameters:
+    -----------
+    eype_name : string
+        a string specifying a edge_type name
+
+    schema : dictionary
+        metadata json object as a dictionary, which is read from the input
+        metadata file from the input dataset
+
+    Returns:
+    --------
+    list : 
+        a list of feature names for a given edge_type
+    """
+    etype_featdict = schema_map[constants.STR_EDGE_DATA]
+    if etype_name in etype_featdict:
+        featnames = []
+        etype_info = etype_featdict[etype_name]
+        for k, v in etype_info.items():
+            featnames.append(k)
+        return featnames
```

Comment:
Use list comprehension to simplify the code. See how @Rhett-Ying did so https://github.com/dmlc/dgl/blob/4a71d35d81d59a7bc65c9afa125cc86cf67b62b8/tools/distpartitioning/utils.py#L70-L90

Reply:
Done. As suggested.